app/models/teleconsultation.rb

```diff
@@ -0,0 +1,42 @@
+class Teleconsultation < ApplicationRecord
```

Comment:
Generic question: what's a good way for us to query how many teleconsultations are incomplete (requested, but never recorded).

Reply:
Currently we could use the `recorded_at` timestamp, Something like:
```
def self.incomplete
  where(`recorded_at IS NOT NULL`)
end
```

Reply:
When we add the `request_completed` flag:
```
def self.incomplete
  where(`recorded_at IS NOT NULL`).or(`request_completed = NULL`)
end 
```